play-frontend-hmrc-play-30/src/main/resources/reference.conf

```diff
@@ -81,3 +81,7 @@ pta-account-menu {
 }
 
 language.fallback.url = "https://www.gov.uk/government/organisations/hm-revenue-customs"
+
+# used to enable A/B testing experiments from the experimentation team
+optimizely.url = "https://cdn.optimizely.com/js/"
```

Comment:
pulled from app-config-common - better to have it here, as the value in app-config-common is actually documented as being there for play-ui so could easily get removed at some point accidentally in the future